.github/workflows/poc-test-environments.yml

```diff
@@ -1,61 +0,0 @@
-# Description: This workflow tests environments split changes for POC purposes.
-#              It will be reverted before changes are merged onto `main`.
```

Comment:
Getting removed from this feature-branch so that `main` doesn't get this. 

But I'll keep testing this feature branch separately in #2661.